docs/get_started/examples/interactive_tensorflow_mnist.rst

```diff
@@ -95,7 +95,7 @@ Install additional dependencies if not already installed
 
 .. code-block:: console
 
-    $ pip install tensorflow==2.8
+    $ pip install tensorflow==2.18.0

```

Comment:
I doubt this will work out of the box. Please verify.

Reply:
removed.